### PR TITLE
Document the href attributes on interfaces including SVGURIReference

### PR DESCRIPTION
**Pull request by foolip**
Follow-up to https://github.com/mdn/content/pull/5111.

### CODE REVIEW

**Comment by foolip** [comment]
There was an impressive diversity of style here, and I tried to match the local style in each file, but probably failed in some case.

File: files/en-us/web/api/svggradientelement/index.html [hunk 1]
```diff
@@ -51,12 +51,14 @@ <h2 id="Properties">Properties</h2>
 <p><em>This interface also inherits properties from its parent, {{domxref("SVGElement")}}.</em></p>
 
 <dl>
+ <dt>{{domxref("SVGGradientElement.href")}} {{ReadOnlyInline}}</dt>
+ <dd>An {{domxref("SVGAnimatedString")}} corresponding to the {{SVGAttr("href")}} or {{SVGAttr("xlink:href")}} attribute of the given element.</dd>
  <dt>{{domxref("SVGGradientElement.gradientUnits")}} {{ReadOnlyInline}}</dt>
- <dd>Returns an {{domxref("SVGAnimatedEnumeration")}} corresponding to the {{SVGAttr("gradientUnits")}} attribute on the given element. Takes one of the constants defined in {{domxref("SVGUnitTypes")}}.</dd>
+ <dd>An {{domxref("SVGAnimatedEnumeration")}} corresponding to the {{SVGAttr("gradientUnits")}} attribute on the given element. Takes one of the constants defined in {{domxref("SVGUnitTypes")}}.</dd>
```
Comment (jpmedley) [inline files/en-us/web/api/svggradientelement/index.html:57]:
```suggestion
 <dd>An {{domxref("SVGAnimatedEnumeration")}} corresponding to the {{SVGAttr("gradientUnits")}} attribute on the given element. This property takes one of the constants defined in {{domxref("SVGUnitTypes")}}.</dd>
```

File: files/en-us/web/api/svggradientelement/index.html [hunk 2]
```diff
@@ -51,12 +51,14 @@ <h2 id="Properties">Properties</h2>
...
  <dt>{{domxref("SVGGradientElement.gradientTransform")}} {{ReadOnlyInline}}</dt>
- <dd>Returns an {{domxref("SVGAnimatedTransformList")}} corresponding to attribute {{SVGAttr("gradientTransform")}} on the given element.</dd>
+ <dd>An {{domxref("SVGAnimatedTransformList")}} corresponding to attribute {{SVGAttr("gradientTransform")}} on the given element.</dd>
```
Comment (jpmedley) [inline files/en-us/web/api/svggradientelement/index.html:59]:
```suggestion
 <dd>An {{domxref("SVGAnimatedTransformList")}} corresponding to the {{SVGAttr("gradientTransform")}} attribute on the given element.</dd>
```

File: files/en-us/web/api/svggradientelement/index.html [hunk 3]
```diff
@@ -51,12 +51,14 @@ <h2 id="Properties">Properties</h2>
...
  <dt>{{domxref("SVGGradientElement.spreadMethod")}} {{ReadOnlyInline}}</dt>
- <dd>Returns an {{domxref("SVGAnimatedEnumeration")}} corresponding to attribute {{SVGAttr("spreadMethod")}} on the given element. One of the spread method types defined on this interface.</dd>
+ <dd>An {{domxref("SVGAnimatedEnumeration")}} corresponding to attribute {{SVGAttr("spreadMethod")}} on the given element. One of the spread method types defined on this interface.</dd>
```
Comment (jpmedley) [inline files/en-us/web/api/svggradientelement/index.html:61]:
```suggestion
 <dd>An {{domxref("SVGAnimatedEnumeration")}} corresponding to the {{SVGAttr("spreadMethod")}} attribute on the given element. One of the spread method types defined on this interface.</dd>
```

File: files/en-us/web/api/svgaelement/index.html [hunk 4]
```diff
@@ -18,8 +18,8 @@ <h2 id="Properties">Properties</h2>
 <dl>
  <dt>{{domxref("SVGAElement.download")}}</dt>
  <dd>See {{domxref("HTMLAnchorElement.download")}}.</dd>
- <dt>{{domxref("SVGAElement.href")}}</dt>
- <dd>See {{domxref("HTMLAnchorElement.href")}}.</dd>
+ <dt>{{domxref("SVGAElement.href")}} {{ReadOnlyInline}}</dt>
+ <dd>Is an {{domxref("SVGAnimatedString")}} that reflects the {{SVGAttr("href")}} or {{SVGAttr("xlink:href")}} attribute.</dd>
```
Comment (jpmedley) [inline files/en-us/web/api/svgaelement/index.html:22]:
```suggestion
 <dd>Aan {{domxref("SVGAnimatedString")}} that reflects the {{SVGAttr("href")}} or {{SVGAttr("xlink:href")}} attribute.</dd>
```

File: files/en-us/web/api/svgfilterelement/index.html [hunk 5]
```diff
@@ -17,6 +17,8 @@
 <h2 id="Properties">Properties</h2>
 
 <dl>
+ <dt>{{domxref("SVGFilterElement.href")}} {{ReadOnlyInline}}</dt>
+ <dd>An {{domxref("SVGAnimatedString")}} that corresponds the {{SVGAttr("href")}} or {{SVGAttr("xlink:href")}} attribute of the given {{SVGElement("filter")}} element.</dd>
```
Comment (jpmedley) [inline files/en-us/web/api/svgfilterelement/index.html:21]:
```suggestion
 <dd>An {{domxref("SVGAnimatedString")}} that corresponds to the {{SVGAttr("href")}} or {{SVGAttr("xlink:href")}} attribute of the given {{SVGElement("filter")}} element.</dd>
```

File: files/en-us/web/api/svggradientelement/index.html (hunk 1 above)
Comment (jpmedley) [inline files/en-us/web/api/svggradientelement/index.html:57]:
```suggestion
 <dd>An {{domxref("SVGAnimatedEnumeration")}} corresponding to the {{SVGAttr("gradientUnits")}} attribute on the given element. This property takes one of the constants defined in {{domxref("SVGUnitTypes")}}.</dd>
```

File: files/en-us/web/api/svgmpathelement/index.html [hunk 6]
```diff
@@ -16,7 +16,12 @@
 
 <h2 id="Properties">Properties</h2>
 
-<p><em>This interface doesn't implement any specific properties, but inherits properties from its parent interface, {{domxref("SVGElement")}}.</em></p>
+<p><em>This interface also inherits properties from its parent, {{domxref("SVGElement")}}.</em></p>
+
+<dl>
+  <dt>{{domxref("SVGMPathElement.href")}} {{ReadOnlyInline}}</dt>
+  <dd>An {{domxref("SVGAnimatedString")}} that corresponds the {{SVGAttr("href")}} or {{SVGAttr("xlink:href")}} attribute of the given {{SVGElement("mpath")}} element.</dd>
```
Comment (jpmedley) [inline files/en-us/web/api/svgmpathelement/index.html:23]:
```suggestion
  <dd>An {{domxref("SVGAnimatedString")}} that corresponds to the {{SVGAttr("href")}} or {{SVGAttr("xlink:href")}} attribute of the given {{SVGElement("mpath")}} element.</dd>
```

**Review by jpmedley — CHANGES_REQUESTED** [review]
Thank you for doing this. I found a few typos.

File: files/en-us/web/api/svgaelement/index.html (hunk 4 above)
Comment (foolip) [reply files/en-us/web/api/svgaelement/index.html:22]: I stuck to the existing convention in this file. I don't like it, but there are quite a lot of or "Is a/an ..." like this across more than 100 files, and I didn't want to fix them all.

**Comment by foolip** [comment]
Thanks @jpmedley, changes applied!

**Review by jpmedley — APPROVED** [review]

**Pull request merged**